json_container/inc/leatherman/json_container/json_container.hpp

```diff
@@ -20,6 +20,9 @@ namespace rapidjson {
 }  // namespace rapidjson
 
 namespace leatherman { namespace json_container {
+    // Constants
+    constexpr size_t DEFAULT_LEFT_PADDING { 4 };
```

Comment:
PR cleanup: this went into the 2nd commit. It should apply to the 1st.

Reply:
You can fix that with `git rebase -i HEAD~2`, then choose `edit` for the older commit and make this change there.

Reply:
Nice, just incorporated those changes. @MikaelSmith 

Does the CHANGELOG also need to be updated or is that after the merge?

Reply:
Add another commit that updates the CHANGELOG and version in CMakeLists.txt.